### ISSUE TITLE
Keyboard Shortcuts

### ISSUE THREAD

**Issue by exurd** [internetarchive/wayback-machine-webextension#966]
It would be nice to have a way to save pages quickly or open archived pages without going into the extension and clicking the corrosponding buttons. It could be "Ctrl+Shift+S" or something similar.

**Comment by cgorringe** [comment]
Thanks, that's a great suggestion!

**Comment by Gitoffthelawn** [comment]
Any progress? This will be a useful addition, as well as an essential improvement for accessibility compliance.

**Comment by CookieSnippets** [comment]
Please, never ever hard code hot keys!
Always make it easily customizable and fully compatible with existing hotkey plugins and internal browser handling.

**Comment by Gitoffthelawn** [comment]
@CookieSnippets Yes, definitely. Just FYI, one of the advantages of Firefox is that it does this automatically. Every extension's keyboard shortcuts can be customized via the "Manage Extension Shortcuts" page within Firefox's Add-ons Manager. In Firefox, you can even customize all the keyboard shortcuts the browser itself uses (e.g. to open a new tab) by going to `about:keyboard`. It's incredibly useful and flexible.
